src/main/java/org/jboss/pnc/bacon/cli/Pig.java

```diff
@@ -45,9 +48,11 @@ public void build(
                     description = "location of the configuration file") String configLocation,
             @Option(
                     names = {"-b", "--build-group"},
-                    defaultValue = Constants.NO_VALUE,
+                    required = true,
```

Comment:
no, it's not required

Reply:
What is the default value then?

Reply:
no value :) will be deduced from build group name in case of a provided build-config.yaml

Reply:
Can it be null?

Reply:
Gotcha

Reply:
Fixed